typings/react-modal.d.ts

```diff
@@ -0,0 +1 @@
+declare module 'react-modal';
```

Comment:
react-modal has its own types. Replace this with this https://www.npmjs.com/package/@types/react-modal.